Makefile

```diff
@@ -4,8 +4,16 @@ OUT := m4rs
 OBJ_DIR := ./obj
 BIN_DIR := ./bin
 
-FLIPS := ./tools/flips.exe
-AS := ./tools/armips-a8d71f0.exe
+ifeq ($(OS),Windows_NT)
+	FLIPS_TMP := ./tools/flips.exe
+	AS_TMP := ./tools/armips-a8d71f0.exe
+else
+	# Assume unix just has it in PATH
+	FLIPS_TMP := flips
+	AS_TMP := armips
+
+FLIPS := FLIPS_TMP
+AS := AS_TMP
```

Comment:
intermediary variables are not necessary, make does not have a concept of scope unless they're defined in a target

Reply:
Ah, my thought was that if you'd want to do something like `make AS=/mie/pee/flips`. but that probably still works.

Reply:
that can be done with `?=` instead